### PR TITLE
Ywt/generate model config

### PR DESCRIPTION
**Pull request by yewentao256**
## Motivation and Context
generate DIOPI model config

## Description
1. call process_ops.py
2. call generate_config.py

## Checklist
**Before PR**:

- [x] I have read and followed the workflow indicated in the [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: diopi_test/python/conformance/model_config/generate_config.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    'linear': dict(atol=1e-3, rtol=1e-4, atol_half=1e-1, rtol_half=1e-2,),
+    'clamp': dict(atol=1e-4, rtol=1e-5,)
+}
+
+tensor_indent = "                    "
+para_indent = "            "
+key_indent = "        "
+seq_name = ['cat', 'stack']
+
+dtype_mappings = {
```
Comment (LeungChiNan) [inline diopi_test/python/conformance/model_config/generate_config.py:48]: image
有的算子需要额外自定义生成数据的函数，防止算子计算的结果含nan

File: diopi_test/python/conformance/model_config/process_ops.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
+import pandas as pd
+import ast
+import re
+
+df = pd.read_csv('dipu_ops.csv', index_col=None)
```
Comment (LeungChiNan) [inline diopi_test/python/conformance/model_config/process_ops.py:5]: 这里不要写死文件名，作为参数传入

File: diopi_test/python/conformance/model_config/process_ops.py [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+        # Append the aggregated shapes for 'tensors' key
+        if row_shapes and 'tensors' in aggregated_params_dict:
+            aggregated_params_dict['tensors']['shape'].extend(row_shapes)
+
+    return f"'{group['diopi_fun'].iloc[0]}': {aggregated_params_dict}"
+
+
+res = df.groupby('diopi_fun').apply(aggregate_rows).tolist()
+
+with open('cv_ops.py', 'w') as f:
```
Comment (LeungChiNan) [inline diopi_test/python/conformance/model_config/process_ops.py:147]: 同上，不写死，实际使用中支持同时保存多个模型测例的中间文件

File: diopi_test/python/conformance/model_config/cv_configs/resnet50_config.py [hunk 4]
```diff
@@ -2,50 +2,81 @@
 from ...diopi_runtime import Dtype
 
 resnet50_config = {
```
Comment (LeungChiNan) [inline diopi_test/python/conformance/model_config/cv_configs/resnet50_config.py:4]: 新工具先不修改这里旧的测例

File: diopi_test/python/conformance/model_config/generate_config.py (hunk 1 above)
Comment (yewentao256) [reply diopi_test/python/conformance/model_config/generate_config.py:49]: 已添加gen_func配置

File: diopi_test/python/conformance/model_config/process_ops.py (hunk 2 above)
Comment (yewentao256) [reply diopi_test/python/conformance/model_config/process_ops.py]: 已改成参数传入

File: diopi_test/python/conformance/model_config/process_ops.py (hunk 3 above)
Comment (yewentao256) [reply diopi_test/python/conformance/model_config/process_ops.py]: 已改成参数传入

File: diopi_test/python/conformance/model_config/cv_configs/resnet50_config.py (hunk 4 above)
Comment (yewentao256) [reply diopi_test/python/conformance/model_config/cv_configs/resnet50_config.py]: 已还原为旧测例

**Review by LeungChiNan — APPROVED** [review]

**Review by yangbofun — APPROVED** [review]

**Pull request merged**